com.microsoft.mrtk.core/Utilities/MathUtilities.cs

```diff
@@ -34,6 +34,55 @@ public static float BytesToMegabytes(ulong bytes)
             return ((float)bytes / (float)Megabyte);
         }
 
+        /// <summary>
+        /// Converts two bytes to one float in the range -1 to 1.
+        /// </summary>
+        /// <param name="a">The least significant byte.</param>
+        /// <param name="b"> The most significant byte.</param>
+        /// <returns>Float value between -1 and 1.</returns>
+        public static float BytesToFloat(byte a, byte b)
```

Comment:
Do we want to write some edit mode tests for these math utilities?

Reply:
we could